### PR TITLE
[WOR-47] Accept Sam tos instead of Cloud Function tos in integration tests

### PR DESCRIPTION
**Pull request by MatthewBemis**
<!--
Hello, friend!
Remember to mention what sort of testing/verification you performed in the course of building this PR, i.e. checking functionality in the browser locally.
Thanks!
--!>

### CODE REVIEW

File: integration-tests/utils/integration-helpers.js [hunk 1]
```diff
@@ -169,7 +169,7 @@ const registerUser = withSignedInPage(async ({ page, token }) => {
   await page.evaluate(async () => {
     await window.catchErrorResponse(async () => {
       await window.Ajax().User.profile.set({ firstName: 'Integration', lastName: 'Test', contactEmail: 'me@example.com' })
-      await window.Ajax().User.acceptTos()
+      await window.Ajax().User.acceptSamTos()
```
Comment (MatthewBemis) [inline integration-tests/utils/integration-helpers.js:172]: ToS lives in Sam now, so call that endpoint instead. The other endpoint is still around due to some backwards compatibility that is necessary for the complex rollout of these changes. Eventually the old ToS code will be removed.

**Review by breilly2 — APPROVED** [review]
👍

**Review by nawatts — APPROVED** [review]

**Review by marctalbott — APPROVED** [review]

**Review by aherbst-broad — APPROVED** [review]

File: integration-tests/utils/integration-helpers.js (hunk 1 above)
Comment (kyuksel) [reply integration-tests/utils/integration-helpers.js:172]: How do we remember to remove the old ToS code when the time comes? Does it make sense to add a comment referencing the future removal to where `Ajax().User.acceptTos()` is defined?

Comment (MatthewBemis) [reply integration-tests/utils/integration-helpers.js:172]: Workspaces team has a ticket in their backlog to clean this up once the old way of handling ToS is fully phased out. See [WOR-153](https://broadworkbench.atlassian.net/browse/WOR-153)

**Pull request merged**